classes/report/fields/runner/coverage/html.php

```diff
@@ -116,6 +116,8 @@ public function __toString()
 
                     if (substr_count($className, '\\') >= 1) {
                         $classTemplate->relativeRootUrl = rtrim(str_repeat('../', substr_count($className, '\\')), DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR;
+                    } else {
+                        $classTemplate->relativeRootUrl = '';
```

Comment:
I think `'./'` might be a little bit more consistent than `''`. Thoughts?

Reply:
It seems `relativeRootUrl` is empty by default; that's why I've used `''`. I've just tested with `./`; it works as well. Lemme know what you prefer, I'll change.

Reply:
I prefer `./` :-).

Reply:
Changed